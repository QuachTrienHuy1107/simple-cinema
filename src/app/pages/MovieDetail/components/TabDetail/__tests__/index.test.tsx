import * as React from "react";
import { render } from "@testing-library/react";

import { TabDetail } from "..";

jest.mock("react-i18next", () => ({
    useTranslation: () => {
        return {
            t: str => str,
            i18n: {
                changeLanguage: () => new Promise(() => {}),
            },
        };
    },
}));

describe("<TabDetail  />", () => {
    it("should match snapshot", () => {
        const loadingIndicator = render(<TabDetail />);
        expect(loadingIndicator.container.firstChild).toMatchSnapshot();
    });
});
