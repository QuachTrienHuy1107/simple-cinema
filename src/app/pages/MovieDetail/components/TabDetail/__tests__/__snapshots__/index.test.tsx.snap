// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabDetail  /> should match snapshot 1`] = `
.c0 {
  position: relative;
}

<div
  class="c0"
>
  <div
    class="ant-alert ant-alert-success ant-alert-no-icon"
    data-show="true"
    role="alert"
  >
    <div
      class="ant-alert-content"
    >
      <div
        class="ant-alert-message"
      >
        Hiện không có lịch chiếu vào ngày này!
      </div>
      <div
        class="ant-alert-description"
      />
    </div>
  </div>
</div>
`;
